requirements.txt

```diff
@@ -2,4 +2,4 @@ pandas
 geopandas
 altair
 requests_html
-unicode
+unicodedata
```

Comment:
So, `unicodedata` is part of the [standard library](https://docs.python.org/3/library/). So I don't think it's needed in the _requirements.txt_ file.